gradle/libs.versions.toml

```diff
@@ -34,12 +34,14 @@ maps-compose = "3.1.1"
 material = "1.11.0-beta01"
 material3-adaptive = "1.0.0-alpha04"
 material3-adaptive-navigation-suite = "1.0.0-alpha02"
+media3Common = "1.2.1"
```

Comment:
I think media3 shares a version number so you should only have one here and use it twice below